### DISCUSSION TITLE
If you learn a foreign language, what would it be?

### DISCUSSION THREAD

**Discussion by Lee-AHCL** [NAIST-SE/AttractingContributors#537] [General]
If you have time to learn a language what would it be? 
For me, I would like to study Russian. 
Any language is fine, dead languages, artificial languages like Esperanto, Klingon, so on... 
But please, no programming languages. 

外国語を勉強すると何を学びたいてすか？
私の場合は、ロシア語を勉強したいと思います。
なんでもよろしいです、死語、エスペラントやクリンゴンみたいな人工言語まで。
しかしプログラミング言語はご遠慮ください。

https://github.com/Lee-AHCL/AttractingContributors-Foreign_Language

**Comment by ani-hovhannisyan** [comment]
Hi, I have added 8th line in your project, please can you do it back to mine?
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git
